home/views.py

```diff
@@ -168,18 +174,27 @@ def update_cwhw(request):
         section_id = request.POST['section']
         section = Section.objects.get(section_id=section_id)
 
+        to_be_taught = request.POST['extra-info-2']
+        subject_taught = request.POST['t']
```

Comment:
Please use some better name to represent the field in the HTML file (talking about `t`).